external

```diff
@@ -0,0 +1 @@
+bazel-out/../../../external
```

Comment:
What is this?

Reply:
Just adds a symbolic link to `external` folder of bazel.

Reply:
Just adds a symbolic link to `external` folder of bazel. See https://github.com/google-research/raksha/tree/compile_commands%40bgogul

Reply:
OK, as long as it works, that's fine.